packages/commons/package.json

```diff
@@ -12,5 +12,7 @@
     "reset": "yarn clean"
   },
   "dependencies": {
+    "@types/uuid": "^8.3.0",
```

Comment:
I would suggest to remove those two uuid because an unique id is auto-generated by feathers.

Reply:
resolved by 0bcdd865854bae3b21873da2fb3d60762f3e19af